### PR TITLE
db/HTMLTableConverterHelper

### PR DESCRIPTION
**Pull request by pawellysy**
Implemented Serialization helper for HTMLTableConverter.

### CODE REVIEW

**Comment by pawellysy** [comment]
@bre1470 how to serialize the tableHTML option?

File: ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+ *
+ *  Functions
+ *
+ * */
+/**
+ * Change the obj of HTMLTableConverter to its Serialized form.
+ * @param obj Object to serialize
+ * @return Serialized object
+ */
+
```
Comment (karolkolodziej) [inline ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts:37]: Unnecessary space, while at the top one is missing.

**Review by karolkolodziej — APPROVED** [review]

File: ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts (hunk 1 above)
Comment (bre1470) [reply ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts:38]: Space, the final frontier... should go to line 16.

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

**Pull request merged**